Accessibilite attributs partages

TuThoThai opened the pull request:
Changements de toutes les parties du profil France pour :
- mettre à jour les parties de NeTEx
- reprendre la sémantique du profil France unifié
- renvoyer vers la partie Accessibilité sur tous les objets contenant un ACCESSIBILITY ASSESMENT et/ou sur des objets détaillés dans cette partie-là (ex. NAVIGATION PATH)

TuThoThai committed:
Changement section attributs partagés

Changements pour la partie Accessibilité du profil France

TuThoThai committed:
Changements de la mention à la partie accessibilité

- Mise en adéquation de l'articulation des parties du profil France
- Renvoi vers la partie Accessibilité pour les attributs partagés (ACCESSIBILITY ASSESMENT)

TuThoThai committed:
Mise à jour de la section dédiée à l'accessibilité

- Mise en adéquation de l'articulation des parties du profil France
- Renvoi vers la partie Accessibilité pour les attributs partagés (ACCESSIBILITY ASSESMENT)

TuThoThai committed:
Révision de la partie sur l'accessibilité

- Mise en adéquation de l'articulation des parties du profil France
- Renvoi vers la partie Accessibilité pour les attributs partagés (ACCESSIBILITY ASSESMENT)

TuThoThai committed:
Révision de la partie sur l'accessibilité

- Mise en adéquation de l'articulation des parties du profil France
- Renvoi vers la partie Accessibilité pour les attributs partagés (ACCESSIBILITY ASSESMENT)

TuThoThai committed:
Révision de la partie sur l'accessibilité

- Mise en adéquation de l'articulation des parties du profil France
- Renvoi vers la partie Accessibilité

nlehuby commented on NeTEx/elements_communs/index.md:
```diff
@@ -1702,62 +1676,52 @@ etc.)
+que seul le champ </span>***<span class="hl">ValidityCondition</span>***
+<span class="hl"> permet de contenir une description textuelle, sous forme d'un
+texte libre susceptible d'être présenté au public en complément des
+indicateurs ci-dessus.</span>
```
Les paragraphes que tu as ajoutés paraphrasent le tableau "AccessibilityLimitation – Élément (objet inclus)" et le petit paragraphe en dessous qui sont quelques lignes plus bas.
Ça me va de garder le même verbatim que dans la partie accessibilité du profil, mais dans ce cas, il vaut peut-être mieux supprimer le tableau présent plus bas.

TuThoThai replied:
Je me suis rendue compte que le tableau n'est pas repris dans la partie Accessibilité du profil... Donc je ne savais pas s'il était mieux de le laisser dans Éléments communs et de prendre le verbatim d'Accessibilité ou de faire l'inverse.
Preneuse de ton avis ! 😄

nlehuby replied:
Discuté aujourd'hui en séance, voici les décisions :
- on garde juste l'intro dans Éléments communs
- on migre les 2 tableaux dans Accessibilité 6.2
- on migre également le paragraphe sur MobilityImpairedAccess dans Accessibilité 6.2

TuThoThai replied:
Changements effectués

TuThoThai committed:
Simplification de la partie accessibilité d'éléments communs

Renvoi vers la partie Accessibilité du profil France

TuThoThai committed:
Détails des évaluation d'accessibilité partagées

Reprise des 2 tableaux provenant de la partie Éléments communs et de la note sur l'attribut MobilityImpairedAccess

TuThoThai committed:
Suppression ancienne image

Suppression pour remplacement

TuThoThai committed:
Update image8

MàJ sur la base du dernier schéma UML

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
 
 ![image](media/image8.svg)
-*Accessibilité des profils précédents*
+*Accessibilité des autres parties du profil France*
```
```suggestion
*Accessibilité dans les autres parties du profil France*
```

prhod replied:
Modifié dans https://github.com/etalab/transport-profil-netex-fr/pull/263/commits/06d2265f501d8b1a5d7781388ba5e047d130072a

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
 part vis-à-vis des besoins d'accessibilité et d'autre part en terme de
 précision sur les équipements et cheminements rencontrés.
 
+<div class="table-title">AccessibilityAssessment – Element (objet inclus)</div>
```
```suggestion
<div class="table-title">AccessibilityAssessment – Élément (objet inclus)</div>
```

prhod replied:
corrigé

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+
+-   ***unknow*** dans tous les autres cas
+
+<div class="table-title">AccessibilityLimitation – Element (objet inclus)</div>
```
```suggestion
<div class="table-title">AccessibilityLimitation – Élément (objet inclus)</div>
```

prhod replied:
corrigé

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+|                     |                             |                        |     |                                                                                                                                                 |
+|---------------------|-----------------------------|------------------------|-----|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Classifi­cation** | **Nom**                     | **Type**               |     | **Description**                                                                                                                                 |
+|                     | ***WheelchairAccess***      | *LimitationStatusEnum* | 1:1 | Indique si l'accès est possible sans fauteuil roulant (codification: ***true***/***false***/***unknow***/***partial***).                        |
```
```suggestion
|                     | ***WheelchairAccess***      | *LimitationStatusEnum* | 1:1 | Indique si on peut s’y déplacer en fauteuil roulant (codification: ***true***/***false***/***unknow***/***partial***).                        |
```

prhod replied:
modification reprise

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+|---------------------|-----------------------------|------------------------|-----|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Classifi­cation** | **Nom**                     | **Type**               |     | **Description**                                                                                                                                 |
+|                     | ***WheelchairAccess***      | *LimitationStatusEnum* | 1:1 | Indique si l'accès est possible sans fauteuil roulant (codification: ***true***/***false***/***unknow***/***partial***).                        |
+|                     | ***StepFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans franchissement de marche ou d'escalier (codification: ***true***/***false***/ ***unknow***/***partial***). |
```
```suggestion
|                     | ***StepFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si on peut s’y déplacer sans franchir de marche ou d’escalier (codification: ***true***/***false***/ ***unknow***/***partial***). |
```

prhod replied:
modification reprise

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+| **Classifi­cation** | **Nom**                     | **Type**               |     | **Description**                                                                                                                                 |
+|                     | ***WheelchairAccess***      | *LimitationStatusEnum* | 1:1 | Indique si l'accès est possible sans fauteuil roulant (codification: ***true***/***false***/***unknow***/***partial***).                        |
+|                     | ***StepFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans franchissement de marche ou d'escalier (codification: ***true***/***false***/ ***unknow***/***partial***). |
+|                     | ***EscalatorFreeAccess***   | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'escalator (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
```
```suggestion
|                     | ***EscalatorFreeAccess***   | *LimitationStatusEnum* | 0:1 | Indique si on peut s’y déplacer sans utiliser d’escalator (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
```

prhod replied:
modification reprise

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+|                     | ***WheelchairAccess***      | *LimitationStatusEnum* | 1:1 | Indique si l'accès est possible sans fauteuil roulant (codification: ***true***/***false***/***unknow***/***partial***).                        |
+|                     | ***StepFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans franchissement de marche ou d'escalier (codification: ***true***/***false***/ ***unknow***/***partial***). |
+|                     | ***EscalatorFreeAccess***   | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'escalator (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
+|                     | ***LiftFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'ascenseur (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
```
```suggestion
|                     | ***LiftFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si on peut s’y déplacer sans utiliser d’ascenseur (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
```

prhod replied:
modification reprise

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+|                     | ***StepFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans franchissement de marche ou d'escalier (codification: ***true***/***false***/ ***unknow***/***partial***). |
+|                     | ***EscalatorFreeAccess***   | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'escalator (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
+|                     | ***LiftFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'ascenseur (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
+|                     | ***AudibleSignsAvailable*** | *LimitationStatusEnum* | 0:1 | Indique si une signalétique auditive est disponible (codification: ***true***/***false***/***unknow***/***partial***).                          |
```
```suggestion
|                     | ***AudibleSignsAvailable*** | *LimitationStatusEnum* | 0:1 | Indique si une information sonore ou une signalétique auditive est disponible (codification: ***true***/***false***/***unknow***/***partial***).                          |
```

prhod replied:
modification reprise

nlehuby commented on NeTEx/accessibilite/index.md:
```diff
@@ -1118,13 +1117,95 @@ texte libre susceptible d'être présenté au public en complément des
+|                     | ***EscalatorFreeAccess***   | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'escalator (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
+|                     | ***LiftFreeAccess***        | *LimitationStatusEnum* | 0:1 | Indique si l'accès est possible sans utiliser d'ascenseur (codification: ***true***/***false***/***unknow***/ ***partial***).                   |
+|                     | ***AudibleSignsAvailable*** | *LimitationStatusEnum* | 0:1 | Indique si une signalétique auditive est disponible (codification: ***true***/***false***/***unknow***/***partial***).                          |
+|                     | ***VisualSignsAvailable***  | *LimitationStatusEnum* | 0:1 | Indique si une signalétique visuelle est disponible (codification: ***true***/***false***/***unknow***/***partial***).                          |
```
```suggestion
|                     | ***VisualSignsAvailable***  | *LimitationStatusEnum* | 0:1 | Indique si une information visuelle ou une signalétique visuelle est disponible (codification: ***true***/***false***/***unknow***/***partial***).                          |
```

nlehuby replied:
Et il manque *TactileGuidanceAvailable* : Indique s'il y a des éléments linéaires au sol qui peuvent être détectés et suivis avec une canne, comme par exemple une bande de guidage podotactile

nlehuby replied:
J'avais déjà fait toutes ces modif de wording dans https://github.com/etalab/transport-profil-netex-fr/pull/218 c'est dommage de ne pas être reparti de là...

prhod replied:
J'ai repris le texte de la PR 218 dans https://github.com/etalab/transport-profil-netex-fr/pull/263.
Les mentions entre parenthèses `(codification: ***true***/***false***/***unknow***/***partial***)` ne sont donc plus présentes.

nlehuby commented on NeTEx/elements_communs/index.md:
```diff
@@ -1700,108 +1674,7 @@ etc.)
-class="hl"> peut être remplie). Les informations textuelles contenues
-doivent pouvoir être présentées au public sans autre
-modification.</span>
+Les informations concernant l'ACCESSIBILITÉ sont utilisées de la même façon pour les LIEUx D'ARRÊT, les LIGNEs,  les COURSEs, etc. L’information d’accessibilité présentée correspond à une information minimale lorsque les ÉVALUATIONs D’ACCESSIBILITÉ (ACCESSIBILITY ASSESSMENT) sont utilisées. Pour plus d'information sur les éléments retenus dans le profil France, se référer à la partie Accessibilit ...
```
```suggestion
Les informations concernant l'ACCESSIBILITÉ sont utilisées de la même façon pour les LIEUx D'ARRÊT, les LIGNEs,  les COURSEs, etc. L’information d’accessibilité présentée correspond à une information minimale lorsque les ÉVALUATIONs D’ACCESSIBILITÉ (ACCESSIBILITY ASSESSMENT) sont utilisées. Pour plus d'information sur les éléments retenus dans le profil France, se référer à la partie Accessibilité du profil France de NeTEx. Cette dernière propose, en outre, des éléments de description d'accessibilité plus détaillés, ainsi que l'identification des attributs obligatoires pour se conformer à la réglementation en vigueur.
```

nlehuby commented on NeTEx/reseaux/index.md:
```diff
@@ -2634,7 +2627,7 @@ D'EMBARQUEMENT (QUAY).
 <td>0:1</td>
 <td><p>Description du cheminement utilisé pour cette correspondance.</p>
-<p><span class="hl">Dans le cadre du Profil Réseau, le NAVIGATION PATH n'est utilisé que pour indiquer de façon générale les contraintes d'accessibilité du cheminement (champs AccessFeatureList et NavigationType). La description complète et détaillée du NAVIGATION PATH n'interviendra que dans un profil dédié.</span></p></td>
+<p><span class="hl">Dans le cadre du Profil Réseau, le NAVIGATION PATH n'est utilisé que pour indiquer de façon générale les contraintes d'accessibilité du cheminement (champs AccessFeatureList et NavigationType). La description complète et détaillée du NAVIGATION PATH est dans la partie Accessibilité du profil France.</span></p></td>
```
Je ne comprends pas trop cette remarque : pourquoi on veut limiter l'utilisation de NAVIGATION PATH ? et le champ AccessFeatureList n'est pas repris dans notre tableau qui décrit les champs de cet objet (table 15, partie accessibilité) :thinking: 

Je remplacerais bien tout le paragraphe par : 

```suggestion
<p>(voir la partie accessibilité du profil pour plus de détails)</p></td>
```

prhod replied:
@nlehuby Je suis d'accord avec toi ! 
corrigé dans https://github.com/etalab/transport-profil-netex-fr/pull/263/commits/06d2265f501d8b1a5d7781388ba5e047d130072a

nlehuby commented on NeTEx/reseaux/index.md:
```diff
@@ -1065,7 +1058,7 @@ de nuit, etc.).
 <td><em>Accessibility­Assessment</em></td>
 <td>0:1</td>
-<td>Information concernant l'accessibilité de la ligne <span class="hl">(</span><em><span class="hl">voir le document </span><strong><span class="hl">Profil NeTEx éléments communs</span></strong></em><span class="hl">).</span></td>
+<td>Information concernant l'accessibilité de la ligne <span class="hl">(</span><em><span class="hl">voir le document </span><strong><span class="hl">Profil NeTEx éléments communs</span></strong>, ainsi que la partie Accessibilité du profil France</em><span class="hl">).</span></td>
```
```suggestion
<td>Information concernant l'accessibilité de la ligne <span class="hl">(<em>voir la partie Accessibilité du profil France</em>).</span></td>
```

prhod replied:
corrigé dans https://github.com/etalab/transport-profil-netex-fr/pull/263/commits/06d2265f501d8b1a5d7781388ba5e047d130072a

nlehuby reviewed (changes requested):
Les descriptions des AccessibilityLimitations ne sont pas conformes à ce qu'on avait validé dans la version 2.3 du profil et il y a quelques petites coquilles à corriger.

TuThoThai commented:
@nlehuby : un grand merci pour ta relecture avisée. Toutes mes excuses de ne pas être repartie de la PR #218 que je n'avais plus en tête ! 
@prhod : si tu peux aider sur les corrections, je suis preneuse

prhod commented:
Cette PR et les corrections à y apporter ont été reprise dans la PR https://github.com/etalab/transport-profil-netex-fr/pull/263
